torch/testing/_internal/common_cuda.py

```diff
@@ -31,6 +31,8 @@
 SM80OrLater = LazyVal(lambda: torch.cuda.is_available() and torch.cuda.get_device_capability() >= (8, 0))
 SM90OrLater = LazyVal(lambda: torch.cuda.is_available() and torch.cuda.get_device_capability() >= (9, 0))
 
+IS_JETSON = LazyVal(lambda: torch.cuda.is_available() and torch.cuda.get_device_capability() in [(7, 2), (8, 7)])
```

Comment:
Is it worthwhile to include 6.2 (Jetson TX2) and 5.3 (Jetson Nano)?

Reply:
This check is mostly for NV / internal CI so the existing check should be good